lib/slack/web/pagination/cursor.rb

```diff
@@ -7,23 +7,31 @@ class Cursor
 
           attr_reader :client
           attr_reader :verb
+          attr_reader :pause
```

Comment:
I feel a bit uncomfortable calling this `pause` because it's unclear whether that's a verb or a noun, maybe `sleep_interval`? Don't feel strongly about this one though.